test/integration/uk/gov/hmrc/exports/repositories/DeclarationRepositoryTest.scala

```diff
@@ -56,7 +56,7 @@ class DeclarationRepositoryTest
       .toInt
 
   private def givenADeclarationExists(declarations: ExportsDeclaration*): Unit =
-    repository.collection.insert(false).many(declarations).futureValue
+    repository.collection.insert(ordered = true).many(declarations).futureValue
```

Comment:
Why is this necessary?

Reply:
yes - test depends on strict order of fetched documents - this change is only in tests